File: 2021/helloworld/sum.js
var readline = require('readline');
var rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
  terminal: false
});

rl.on('line', function(line){
    let nums = line.split(' ').map(n => parseInt(n));
    console.log(nums[0] + nums[1]);
});

File: 2021/helloworld/sum.rs
use std::io;

fn main() {
    let mut line = String::new();
    io::stdin()
        .read_line(&mut line)
        .expect("failed to read from stdin");
    let mut nums = line.split(' ');
    let a: i64 = nums.next().unwrap().trim().parse().unwrap();
    let b: i64 = nums.next().unwrap().trim().parse().unwrap();
    println!("{:?}", a + b);
}
